apps/prairielearn/package.json

```diff
@@ -63,7 +63,7 @@
     "bowser": "^2.11.0",
     "byline": "^5.0.0",
     "chalk": "^4.1.2",
-    "cheerio": "^0.22.0",
+    "cheerio": "^1.0.0-rc.12",
```

Comment:
You'll want to remove `@types/cheerio` as well since the latest version ships with its own types.